### DISCUSSION TITLE
Option to automatically switch between single/double page mode depending on device orientation

### DISCUSSION THREAD

**Discussion by radonmiser** [gotson/komga#1532] [Ideas]
### Describe your suggested feature

in the default web reader add an option to automatically switch to single page mode when the device is rotated to portrait mode, and switch back to double page mode when the device is rotated to horizontal mode.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by floatiepen** [comment]
I was looking for this feature too!  it would be great!
